core/models.py

```diff
@@ -97,3 +97,18 @@ class Card(models.Model):
 
     def __str__(self):
         return "%s - %s (%s)" % (self.user.username, self.card_number, self.comment)
+
+
+class NfcCard(models.Model):
```

Comment:
Jeg synes i hvert fall dette er samme modell, og at man igrunn kompliserer det ved å ha to modeller. Vi snakker tross alt om "kort" her, enten man finner det frem med UID eller kortnummer..